ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -241,9 +246,13 @@ - (void)capturePressed {
                 // Hide the template image after capturing
                 _previewView.templateImageHidden = YES;
         
-                // Reset the continue button title and configure the skip button as a recapture button
-                _continueSkipContainer.continueButtonItem.title = self.continueTitle;
-                _continueSkipContainer.skipButtonItem = _skipButtonItem;
+                // If we experienced an error during the capture (likely writing the file to disk)
+                // then do not reconfigure the buttons
+                if (!self.error) {
+                    // Reset the continue button title and configure the skip button as a recapture button
+                    _continueSkipContainer.continueButtonItem.title = self.continueTitle;
+                    _continueSkipContainer.skipButtonItem = _skipButtonItem;
```

Comment:
Again, want to handle both cases properly. Just factor it out into a method that checks self.error and does the right thing.